albumentations/augmentations/functional.py

```diff
@@ -795,8 +795,10 @@ def add_shadow(img, vertices_list):
     return image_rgb
 
 
-def invert(img):
-    return 255 - img
+def invert(img: np.ndarray) -> np.ndarray:
+    # Supports all the valid dtypes
+    # clips the img to avoid unexpected behaviour.
+    return MAX_VALUES_BY_DTYPE[img.dtype] - clip(img, img.dtype, MAX_VALUES_BY_DTYPE[img.dtype])
```

Comment:
We do not need clip there

Reply:
I thought maybe if the user enters an array that is not of valid floats, e.g. floats not in range. But I'm guessing there are some tests in place for that already and it's better to yield an exception. Thank you for the feedback